server/src/main/java/org/elasticsearch/ElasticsearchException.java

```diff
@@ -384,6 +374,17 @@ protected static void innerToXContent(XContentBuilder builder, Params params,
         if (params.paramAsBoolean(REST_EXCEPTION_SKIP_STACK_TRACE, REST_EXCEPTION_SKIP_STACK_TRACE_DEFAULT) == false) {
             builder.field(STACK_TRACE, ExceptionsHelper.stackTrace(throwable));
         }
+
+        Throwable[] allSuppressed = throwable.getSuppressed();
+        if (allSuppressed.length > 0) {
+            builder.startArray(SUPPRESSED.getPreferredName());
+            for (Throwable suppressed : allSuppressed) {
+                builder.startObject();
+                generateThrowableXContent(builder, params, suppressed);
+                builder.endObject();
+            }
+            builder.endArray();
+        }
```

Comment:
out of curiosity, does order matter here? Why moving suppressed?

Reply:
I think it is easier to read when the suppressed comes after the headings. That keeps everything about each exception close together. To be honest, if it is important to me, I should add a unit test for it. I'll do that.